### ISSUE TITLE
Iconified windows are hard to  drag

### ISSUE THREAD

**Issue by rickmcgeer** [LivelyKernel/lively.next#110]
When I iconify a window, it collapses nicely, but the pointer needs to be very close to the edge to drag it anywhere.

**Comment by linusha** [comment]
This is a side effect of 81ff17d24ce586367cfe6a78b90fc5bdd201c110, which introduced 'onDoubleMouseDown' on the title to maximize the window.
I totally agree with you, that the movement of minimzed windows is impractical now.

I will take a look at this hopefully later today.

**Comment by linusha** [comment]
I do not see a way to keep the current functionality of toggling maximize when double clicking the label and preserve the ability to drag a window around, even if it is "grabbed" at the label. Maybe @merryman sees one?

If not, I would think the dragging is more important and therefore remove the doubleClick on the title?
I still really like to be able to maximize a window with one click, to the point that I think it would be cool to have a fourth button in the top left corner that achieves what currently the dblClick on the title allows, e.g. using [this](https://fontawesome.com/icons/expand-arrows-alt?style=solid). However, that would be quite invasive to the user experience and I am not sure if anyone besides me would be so eager to have this option?

**Comment by merryman** [comment]
So I just pushed 3eac31340fe3e4b174d4e1a103be781dfbe6c37b which somewhat fixes the issue.
I realized that maximizing out of a minimized window is broken anyways (the targetMorph is invisible) so I assumed that was not part of the desired semantics anyways?
As of now, maximization toggle only works when the window has not been collapsed. That means we can stop the label from listening to mouse events in that state.
If that is ok to you @linusha let us keep it like that.
If we find us to really want to maximize collapsed windows, we need to start synthesizing the event by checking for bounds and position of the mouse event manually.
